packages/components/src/components/text/text.jsx

```diff
@@ -11,10 +11,13 @@ const Text = ({ children, size, color, align, weight, line_height, as, className
     };
 
     const text_align = align || 'left';
```

Comment:
```suggestion
    const text_align = align || 'start';
```

Suggested default value as `start` because we might need to cater for `ltr` and `rtl` situations in the future (if we end up supporting languages that are `rtl`, and it's better to follow the current CSS standards.

Please refer to [here](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align) for default values.